conversations/v3/whatsapp.yaml

```diff
@@ -2522,7 +2520,7 @@ components:
           type: string
           description: The phone number
           example:
-            49123456789
+            "49123456789"
```

Comment:
Not sure here. Is this correct?

Reply:
the example don't need the " ,IIRC

Reply:
ATM, it is rendered as a number. Is it a number? Otherwise, I think it needs the ". See https://api.tyntec.com/reference/conversations/current.html#whatsapp-whatsappphonenumber :

> **WhatsAppPhoneNumber**
> ```json
> {
>  "phoneNumber": 4912345678,
>  ...
>}
>```